src/controllers/creditCard.controller.ts

```
@@ -25,10 +25,40 @@ export const getCurrentUserCard = catchAsync(async (req: Request, res: Response,
   }
 
   const creditCard = await getCreditCardByUserId(req.user._id);
-  
+
+  // Lấy danh sách bank từ API vietqr
+  let bankList: any[] = [];
+  try {
+    const axios = require('axios');
+    const response = await axios.get('https://api.vietqr.io/v2/banks');
+    if (response.data && response.data.data) {
+      bankList = response.data.data;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)